cmd/util/trace.go

```diff
@@ -0,0 +1,58 @@
+package util
```

Comment:
`cmd/util` is just for command related utilities. 
Please move this file under the `util/argo` package

Reply:
Moved.